httpbin/cmd/cmd_test.go

```diff
@@ -532,6 +595,13 @@ func TestMainImpl(t *testing.T) {
 				t.Fatalf("expected return code %d, got %d", tc.wantCode, gotCode)
 			}
 
+			if tc.regex {
+				if _, err := regexp.MatchString(tc.wantOut, out); err != nil {
```

Comment:
This test doesn't work, we need to check the currently-ignored `matched` bool returned by regexp.MatchString.  When you do that, you'll see that the tests fail.

I think a better approach might be to have a separate `wantOutFn func(out string) bool` entry in the testcase struct, and for these couple of "regex" tests instead check for specific substrings?

Reply:
Yeah I agree but I kinda lost about which texts I should actually check in sepreate `wantOutFn`. Could you please give me some example for that?

Reply:
I pushed my changes about this one as well.